### PR TITLE
Renaming to Sequential Coimits

### PR DESCRIPTION
**Pull request by kangrongji**
See #1046 .

I also changed some names in `SequentialColimit/Properties.agda` for it being more informative. These names are only used in this one file. Also add some spaces to make them look better.

@felixwellen

### CODE REVIEW

File: Cubical/HITs/James/Inductive/PushoutFormula.agda [hunk 1]
```diff
@@ -26,7 +26,7 @@ open import Cubical.Data.Sigma
 open import Cubical.HITs.Wedge
 open import Cubical.HITs.Pushout
 open import Cubical.HITs.Pushout.PushoutProduct
-open import Cubical.HITs.SequentialColimit
+open import Cubical.HITs.SequentialColimit renaming (incl to incl')
```
Comment (felixwellen) [inline Cubical/HITs/James/Inductive/PushoutFormula.agda:29]: I guess this is due to a name collision. I would prefer a name which clearly connects ```incl``` to ```SeqColim```, so it is clear on reading where it comes from.
Sometimes it is good to ```import``` something as, say ```SColim``` to later write ```SColim.incl```, but I don't know if it is a good idea here.

**Review by felixwellen — CHANGES_REQUESTED** [review]
Thanks! Just one comment so far. Please keep it as a draft as long as you are still adding commits.

File: Cubical/HITs/James/Inductive/PushoutFormula.agda (hunk 1 above)
Comment (kangrongji) [reply Cubical/HITs/James/Inductive/PushoutFormula.agda]: Make sense. This is the only place where conflicting happens, I'll follow you here.

**Comment by kangrongji** [comment]
It's done now @felixwellen

**Review by felixwellen — APPROVED** [review]
Looks all good!

**Pull request merged**